yt/utilities/lib/bounding_volume_hierarchy.pxd

```diff
@@ -1,3 +1,4 @@
+# cython: legacy_implicit_noexcept=True
```

Comment:
This generates a bunch of warnings so it's not to be considered stable, but it's useful to quickly slap `noexcept` on all functions in the module and reduce the difference between the old cythonized C code and the new one. I should also point out that it's not enough to solve the performance issue seen in the example I'm currently working on.